srcpkgs/singularity/INSTALL

```diff
@@ -0,0 +1,9 @@
+case "${ACTION}" in
+post)
+	if [ ! -d var/lib/singularity/mnt/session ]; then
+		mkdir -p var/lib/singularity/mnt/session
+	fi
+	chown root:root var/lib/singularity/mnt/session
+	chown root:users usr/libexec/singularity/bin/starter-suid
+	chmod 4755 usr/libexec/singularity/bin/starter-suid
```

Comment:
Shouldn't this be 4750? That's what xbps-uchroot uses, at least.

Reply:
4755 is the upstream default but 4750 works fine as well.
In that case, do we also want to add a `singularity` group to further restrict access?

Reply:
I assume you want 4750 since you made it belong to the `users` group, otherwise chown'ing it makes no difference that I can see.

And yes, it's likely that a `_singularity` group would be best.

Reply:
Changed.

Reply:
Btw, `chmod`'ing to 4750 in `do_install` is also necessary, otherwise there will be a window of time where the binary is 4755 and anyone can run anything.

Reply:
Fixed.